src/views/summeruniversity/Single.vue

```diff
@@ -429,8 +435,14 @@ export default {
       this.event = response.data.data
       this.can = response.data.permissions
 
-      this.event.start_city = this.event.locations.find(location => location.start === true).name
-      this.event.end_city = this.event.locations.find(location => location.end === true).name
+      const startLocation = this.event.locations.find(location => location.start === ('true' || true))
```

Comment:
Why checking if it's a string? Can there be cases like these?

Reply:
Unfortunately I seems that both are in the database now. I will fix this when I fix the issue with the false value being stored

Reply:
I'd rather update the DB, then this won't be needed, what do you think?

Reply:
That might be good. Can you change them all to the same type?

Reply:
I've changed it to the boolean true. If it's okay with you @serge1peshcoff I would like to merge this already and we can change the DB in the coming days. Is that fine?

Reply:
I think so